system/doc/design_principles/statem.xml

```diff
@@ -1693,12 +1693,12 @@ open(cast, {button,_}, Data) ->
     <p>
       If you need to cancel a timer because of some other event, you can use
       <seealso marker="erts:erlang#cancel_timer/2"><c>erlang:cancel_timer(Tref)</c></seealso>.
-      Note that a time-out message cannot arrive after this,
-      unless you have postponed it before (see the next section),
+      Note that no time-out message will arrive after this (because the timer has been explicitly canceled),
+      unless you have already postponed one earlier (see the next section),
```

Comment:
I think your text is not completely clear about the fact that there is only one time-out message.
How about your text, but: 
> Note that **the** time-out message will **not** arrive ...
> ...already postponed **it** earlier ...

Reply:
Yes, it was indeed ambiguous. Took your suggestion, and made it even more explicit below. What do you think?

> Note that **the canceled timer's** time-out message will **not** arrive after this (because the timer has been explicitly canceled), unless you have already postponed **it** earlier

Reply:
Also, "_timeout_" is used in code examples, and "_time-out_" in the surrounding texts, but there are deviations. I was thinking about making them consistent.

Reply:
That looks good, but, is the "(because ... canceled)" parenthesis still needed? Or does it only open for an ambiguity about if the following **it** refers to the time-out message or to the the timer?

Reply:
We had technical writers that introduced "time-out", which should be used in the text, but in code the atom to use is `timeout`.  Deviations should be corrected.